test/files/run/splain.check

```diff
@@ -0,0 +1,62 @@
+newSource1.scala:13: error: implicit error;
```

Comment:
I find `implicit error` very terse. Also the default message says upfront what the target parameter is. Could we keep `could not find implicit value for parameter X: T`?